src/app/Main/Header/Export/Dialog/Header-ExportDialog.component.tsx

```diff
@@ -1,34 +1,31 @@
 import React from "react";
 import { DialogContentText } from "@material-ui/core";
 import { useTranslation } from "react-i18next";
-import { DialogComponent } from "../../../Shared/Dialog.component";
-import { FilterContextInterface } from "../../../Shared/Context/FilterContext";
-import { ExportInterface } from "../../../Shared/Model/Export.model";
-import { HeaderExportButtonComponent } from "./Header-ExportButton.component";
-import { ExportDialogCheckboxesComponent } from "./Dialog/ExportDialog-Checkboxes.component";
-import { ExportDialogButtonsComponent } from "./Dialog/ExportDialog-Buttons.component";
-import { ExportButtonLabelComponent } from "./Export-ButtonLabel.component";
-import { ExportLabels } from "./ExportServices/generateExportLabels.service";
-import { ExportDialogWarningComponent } from "./Dialog/ExportDialog-Warning.component";
+import { DialogComponent } from "../../../../Shared/Dialog.component";
+import { FilterContextInterface } from "../../../../Shared/Context/FilterContext";
+import { ExportInterface } from "../../../../Shared/Model/Export.model";
+import { ExportDialogCheckboxesComponent } from "./ExportDialog-Checkboxes.component";
+import { ExportDialogButtonsComponent } from "./ExportDialog-Buttons.component";
+import { ExportLabels } from "../ExportServices/generateExportLabels.service";
+import { ExportDialogWarningComponent } from "./ExportDialog-Warning.component";
 
 export function HeaderExportDialogComponent(props: {
     isOpen: boolean;
     settings: ExportInterface;
     exportLabels: ExportLabels;
+    buttonLabel: JSX.Element;
```

Comment:
The ButtonLabel is created in the container because it is passed to two children components. I would like to leave it as it is. Changes will then come in a header refactoring (new ticket).

Reply:
Yes, but you can create one label hear and one in the container, that would be no problem. But it's fine to leave it as it is.